server/cmd/arkd/main.go

```diff
@@ -18,19 +22,40 @@ var (
 	date    = "unknown"
 )
 
-func main() {
+// flags
+var (
+	urlFlag = &cli.StringFlag{
+		Name:  "url",
+		Usage: "the url where to reach ark server",
+		Value: fmt.Sprintf("http://localhost:%d", config.DefaultPort),
+	}
+	macaroonFlag = &cli.StringFlag{
+		Name:  "macaroon-path",
+		Usage: "the path where to find the admin macaroon file",
+		Value: filepath.Join(common.AppDataDir("arkd", false), "macaroons", "admin.macaroon"),
+	}
+	tlsCertFlag = &cli.StringFlag{
+		Name:  "tls-cert-path",
+		Usage: "the path where to find the TLS certificate",
```

Comment:
Can we make it so if not passed it assume its' unsecure? (ideally because we have a proxy that does TLS termination already ie. coolify)

Reply:
Certainly, and it's already supported! If the URL starts with `http://` this flag is just ignored.

In general, the CLI understands by itself if the tls cert is required or not depending on the given url